SimulationRuntime/c/util/rtclock.c

```diff
@@ -402,6 +402,23 @@ double rt_ext_tp_tock(rtclock_t* tick_tp) {
   return d - min_time;
 }
 
+void rt_ext_tp_tick_realtime(rtclock_t* tick_tp) {
+  *tick_tp = mach_absolute_time();
+}
+
+double rt_ext_tp_tock_realtime(rtclock_t* tick_tp) {
+  double d=0;
+  rtclock_t tock_tp = mach_absolute_time();
```

Comment:
This would need to use `mach_timebase_info()` to return the delta in seconds between the two values.